pkg/cmd/extension/ext_tmpls/buildScript.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+echo "TODO implement this script."
+echo "It should build binaries in /dist/<platform>-<arch>[.exe] as needed."
```

Comment:
Nit: maybe remove the leading slash from `/dist`? Otherwise someone might interpret that as an absolute path